content/en/docs/application-store/external-apps/deploy-clickhouse.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere"
+keywords: 'KubeSphere, Kubernetes, ClickHouse, ClickHouse Operator, ClickHouse Cluster'
+description: 'Learn how to deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere.'
+linkTitle: "Deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere"
+weight: 14340
+---
+
+[ClickHouse](https://clickhouse.tech/) is a column-oriented database management system (DBMS) for online analytical processing of queries (OLAP). ClickHouse on QingCloud is a ClickHouse cluster application deeply customized with the following features.
+
+- Data redistribution in clusters. Apart from automated management run in the cluster, you can also perform cluster-level data operations, including SYSTEM RESHARDING, SYSTEM RESHARDING DATABASE [database_name], and SYSTEM RESHARDING TABLE [database_name].[table_name]. Since you can redistribute your cluster data based on you needs, data query performance will be evidently optimized with less scenarios like inappropriate data distribution.
+- Excellent performance and less cost. ClickHouse on QingCloud adjusts certain key parameters and carries the excellent performance of ClickHouse clusters over into both single-node and multi-node cluster environments with various disk types supported. You can choose the disk type based on your needs and experience the excellent performance with no need to adjust any parameters.
+
+This tutorial demonstrates how to deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere.
+
+## Prerequisites
+
+- You need to enable [the OpenPitrix system](../../../pluggable-components/app-store/).
+- You need to create a workspace, a project, and two user accounts (`ws-admin` and `project-regular`) for this tutorial. The account `ws-admin` must be granted the role of `workspace-admin` in the workspace, and the account `project-regular` must be invited to the project with the role of `operator`. This tutorial uses `demo-workspace` and `demo-project` for demonstration. If they are not ready, refer to [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/).
+
+## Hands-on Lab
+
+### Step 1: Deploy ClickHouse Operator
+
+1. Log in to the KubeSphere Web console as `admin`, and use **Kubectl** from the **Toolbox** in the bottom right corner to run the following command to install ClickHouse Operator.
+
+   ```bash
+   kubectl apply -f https://raw.githubusercontent.com/TCeason/clickhouse-helm-chart/main/clickhouse-operator-install.yml
```

Comment:
We should not use a personal GitHub account in the documentation.

Reply:
@Felixnoo Use this link https://raw.githubusercontent.com/radondb/radondb-clickhouse-kubernetes/master/clickhouse-operator-install.yml